blog/en/blog/2023/02/09/secure-apisix-admin.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Securing Admin Access to Apache APISIX
+authors:
+  - name: Nicolas Fränkel
+    title: Author
+    url: https://github.com/nfrankel
+    image_url: https://avatars.githubusercontent.com/u/752258
+keywords:
+  - API gateway security
+  - Admin API
+  - how to secure API gateway
+description: >
+  API Gateways are critical components in one's infrastructure. If an attacker could change the configuration of routes, they could direct traffic to their infrastructure. Consequences could range from data theft to financial losses. Worse, data theft could only be noticed after a long time by mirroring the load. Hence, protecting your API Gateway is of utmost importance
+tags: [Ecosystem]
+image: https://blog.frankel.ch/assets/resources/secure-apisix-admin/security-g85f9e284f.jpg
+---
+
+>API Gateways are critical components in one's infrastructure. If an attacker could change the configuration of routes, they could direct traffic to their infrastructure. Consequences could range from data theft to financial losses. Worse, data theft could only be noticed after a long time by mirroring the load. Hence, protecting your API Gateway is of utmost importance.
```

Comment:
The quote shows up properly, but it violates markdown syntax, you should add a space between > and quoted texts, see https://www.markdownguide.org/basic-syntax/#blockquotes-1

```suggestion
> API Gateways are critical components in one's infrastructure. If an attacker could change the configuration of routes, they could direct traffic to their infrastructure. Consequences could range from data theft to financial losses. Worse, data theft could only be noticed after a long time by mirroring the load. Hence, protecting your API Gateway is of utmost importance.
```